src/runtime/contrib/random/mt_random_engine.cc

```diff
@@ -134,12 +134,16 @@ class RandomEngine {
 
  private:
   void FillData(DLTensor* tensor, int64_t size) {
-    // Make the value be 1.0 - 10.0, not (0.0 - 1.0) so that we could satisfy
+    // Make the value be 17.0 - 30.0, not (0.0 - 1.0) so that we could satisfy
     // quantized dtype (uint8 / int8) data non-empty requirement
-    std::uniform_real_distribution<> dist(1.0, 10.0);
+    // We start from 17.0 because two int4 are packed in a single uint8
+    std::uniform_real_distribution<> dist(17.0, 30.0);
```

Comment:
Why 17.0-30.0?

Reply:
17.0 is lower bound for both int4x2 to be > 0, 30.0 is arbitrary upper bound

Reply:
It makes sense to int4. But I'm not sure if it will influence other workloads (e.g int8).

Reply:
edited so that only int uses a different distribution from 17-30